### DISCUSSION TITLE
Zillow gets detected

### DISCUSSION THREAD

**Discussion by quentinjs** [ultrafunkamsterdam/undetected-chromedriver#1151] [General]
The first time it worked!!!! yay.  

But then when i reran it I get the captcha page.  

            #url = 'https://www.zillow.com/homes/' + street_address
            url = 'https://www.zillow.com'
            driver2.get(url)

Thoughts?  Oh your page worked.  It said I passed.

**Comment by quentinjs** [comment]
I solved my problem (for now).  Need to remove cookies, so I just delete the entire user path.  

def clean_cookies(folder_path):

    #folder_path = os.path.join(os.getcwd(), 'selenium')

    print("Remove Folder:", folder_path)
    for root, dirs, files in os.walk(folder_path, topdown=False):
        for file in files:
            file_path = os.path.join(root, file)
            try:
                os.remove(file_path)
                #print(f"Deleted file: {file_path}")
            except OSError as e:
                print(f"Error: {file_path} : {e.strerror}")
        for dir in dirs:
            dir_path = os.path.join(root, dir)
            try:
                shutil.rmtree(dir_path)
                #print(f"Deleted folder: {dir_path}")
            except OSError as e:
                print(f"Error: {dir_path} : {e.strerror}")

##################################################

    sel_path = os.path.join(os.getcwd(), 'selenium')
    clean_cookies(sel_path)
    driver = uc.Chrome(user_data_dir=sel_path)
##################################################

**Comment by quentinjs** [comment]
Weird, it worked as well the first time and then failed afterwards even though the folder was empty.

So back to needing your guidance.

**Reply by markthepixel** [reply]
ever get this working?
